src/main/java/calculator/utils/Splitter.java

```diff
@@ -0,0 +1,21 @@
+package calculator.utils;
+
+public class Splitter {
+    private static final String EMPTY_SPACE_DELIMITER = " ";
+
+    private Splitter() {
```

Comment:
```suggestion
    private Splitter() {}
```
생성자를 막기위한 private 생성자의 경우 한줄로 쓰셔도 괜찮습니다 🙇